From aaaee7b55c9e58410abcef6e3e2fd80e4135d02e Mon Sep 17 00:00:00 2001
From: Will Deacon <will@kernel.org>
Date: Thu, 7 Jul 2022 11:25:40 +0100
Subject: [PATCH] docs: perf: Include hns3-pmu.rst in toctree to fix 'htmldocs'
 WARNING
Git-commit: aaaee7b55c9e58410abcef6e3e2fd80e4135d02e
Patch-mainline: v6.0-rc1
References: bsn#12
Modified-by-SEL: Yes, modified due to different context


After commit 39915b6b5fc2 ("drivers/perf: hisi: Add description for HNS3
PMU driver"),building the 'htmldocs' target results in the following
warning:

 | Documentation/admin-guide/perf/hns3-pmu.rst: WARNING: document isn't included in any toctree

Add 'hns3-pmu' to the perf toctree to silence the warning.

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Will Deacon <will@kernel.org>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 Documentation/admin-guide/perf/index.rst |    1 +
 1 file changed, 1 insertion(+)

--- a/Documentation/admin-guide/perf/index.rst
+++ b/Documentation/admin-guide/perf/index.rst
@@ -8,6 +8,7 @@ Performance monitor support
    :maxdepth: 1
 
    hisi-pmu
+   hns3-pmu
    imx-ddr
    qcom_l2_pmu
    qcom_l3_pmu
